contrib/ci/run

```diff
@@ -58,6 +58,13 @@ declare BASE_DIR=`pwd`
 declare MODERATE=false
 declare RIGOROUS=false
 
+# pep8 was renamed to pycodestyle
+declare PEP8_BIN="pep8"
+which pycodestyle-2 &> /dev/null
```

Comment:
Hi,

this does not work if `pycodestyle-2` is not installed since the script sets `errexit` at the beginning and if ``pycodestyle-2` is not installed `which` returns a error and the script terminates.

Btw, why are you using `pycodestyle-2`? Since python2 is somewhat deprecated in Fedora I think it would be sufficient to only install `python3-pycodestyle` and only look for `pycodestyle-3`.

bye,
Sumit

Reply:
We are running tests with Python2 so imho we should use python2 pep8. But I switch it to python3 only since we will have to run the tests with python3 sooner or later anyway. I'm not sure the tests require any changes for python3, I will add it to my todo list.